<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifMedicament</class>
 <widget class="QDialog" name="DialogModifMedicament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>722</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>722</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modification des médicaments</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mesImages.qrc">
    <normaloff>:/images/leLogo</normaloff>:/images/leLogo</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(123,202,255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Dépot légal :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditDL">
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Nom commercial :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditNC">
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Famille :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxFamilleModif">
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Prix échantillon :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditPrix">
             <property name="styleSheet">
              <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="mesImages.qrc">:/images/leLogo</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="3">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Effets :</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Composition :</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Contre-indications :</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="textEditEffets">
           <property name="styleSheet">
            <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="textEditComposition">
           <property name="styleSheet">
            <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="textEditCI">
           <property name="styleSheet">
            <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:inset;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:outset;</string>
         </property>
         <property name="text">
          <string>&amp;Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAnnuler">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
border-width:1px;
border-color: rgb(21, 165, 255);
border-style:outset;</string>
         </property>
         <property name="text">
          <string>&amp;Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mesImages.qrc"/>
 </resources>
 <connections/>
</ui>
